cmd/jujud/reboot/reboot.go

```diff
@@ -21,6 +23,8 @@ import (
 	"github.com/juju/juju/container"
 	"github.com/juju/juju/container/factory"
 	"github.com/juju/juju/instance"
+	"github.com/juju/juju/service"
+	"github.com/juju/juju/service/common"
 )
 
 var logger = loggo.GetLogger("juju.cmd.jujud.reboot")
```

Comment:
This seems to violate the no global variables mandate. However, it predates these changes so not sure it merits an update in this PR.